GoogleChromeLabs/ps-analysis-tool
# Fix: Update UI/UX of components

## Description
This PR updates few items
- In the dark mode, the “i” icon not visible properly for undermined cookies. 
- Add a hover state for the known breakages. The accordion menu doesn’t have a hover state.
- Upon keyboard navigation to the extension, pressing the Enter key should trigger the collapse/expand functionality of the section.
- Truncate the table cell's value if the content is bigger than the column size.



## Relevant Technical Choices



## Testing Instructions
- Open the Privacy Sandbox panel and navigate to the Cookie landing page.
- Find the known breakages section and hover over the accordion.
- The hover state will change the bg color.
- Now navigate to the cookie table and enable the mapping column.
- You'll find an info icon with Undetermined cookies visible in dark mode.
- Now try resizing some columns, you'll see column values bigger than column size will be truncated.
- Now click on the cookies sidebar title, and try pressing the `Enter` or `Space` key.
- It should toggle the state of the sidebar to open or close.


## Additional Information:



## Screenshot/Screencast


https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/99582722-3a82-4d13-a042-af09f8dc9a07





---

## Checklist






Fixes #
